src/extension/utils/createConfFile.ts

```diff
@@ -152,7 +152,7 @@ export function newFormToConf(newForm: NewForm): string {
 
   if (newForm.specObj.specFile && newForm.solidityObj.mainContract) {
     // verify is an array in the old version, string in the new one
-    const oldCliVersion = false
+    const oldCliVersion = cliVersion === 1
 
     if (oldCliVersion) {
```

Comment:
better verify if-cvl1, else if-cvl2 else-error,

Reply:
this is just to write the --verify flag into the conf file, from what the user chose in the settings form
we decide if we want it to be written as a string or an array with one string
there is no error if there is no cli, it just rights it a an array with a string (default)
it's because you don't need cli to wrote from the setting ui to a conf file, in format